browser/css/jsdialogs.css

```diff
@@ -359,7 +359,7 @@ algned to the bottom */
 }
 .notebookbar.ui-overflow-group:not(.ui-overflow-group-container-with-label) {
 	align-content: center;
-	height: 4.25rem !important;
+	height: 4rem !important;
```

Comment:
it is causing issues as explained in https://github.com/CollaboraOnline/online/pull/12998